.github/workflows/ci.yaml

```diff
@@ -0,0 +1,148 @@
+name: CI Tasks for Ory CLI
+on:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - '*'
+  pull_request:
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17'
+      - run: |
+          make test
+
+  docs:
+    name: Generate docs
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/master' }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17'
+      - run: |
+          mkdir -p docs/docs
+          make docs/cli
+          echo 'docsdir='"./docs" >> $GITHUB_ENV
+      - uses: actions/checkout@v2
+        with:
+          repository: ory/docs
+          path: ${{ env.docsdir }}
+          token: ${{ secrets.PAT }}
```

Comment:
Can we scope the permissions needed for this token?

Reply:
Same as above.

Reply:
Thanks! To ensure we do not have any collisions, could you rename this secret to `ORY_BOT_PAT`? :)